VERSION

```diff
@@ -1 +1 @@
-0.7.2-alpha.0
\ No newline at end of file
+0.7.2-alpha.1
```

Comment:
I assume this is for testing release workflow?

Reply:
Yup. I'm doing a pre-release test now.